### PR TITLE
Fix #16552 - White on gray text on check table status results

### LINKED ISSUE #16552: White on gray text on check table status results
When checking a table

5.2.0-dev

image

### PR DESCRIPTION
**Pull request by ujjwal-batra**
Signed-off-by: Kumar Ujjwaldisha Batra <ujjwal24batra@gmail.com>

### Description

Fixes grey checkboxes when checked.

Fixes #16552 

Before submitting pull request, please review the following checklist:

- [x] Make sure you have read our [CONTRIBUTING.md](https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md) document.
- [x] Make sure you are making a pull request against the correct branch. For example, for bug fixes in a released version use the corresponding QA branch and for new features use the _master_ branch. If you have a doubt, you can ask as a comment in the bug report or on the mailing list.
- [x] Every commit has proper `Signed-off-by` line as described in our [DCO](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO). This ensures that the work you're submitting is your own creation.
- [x] Every commit has a descriptive commit message.
- [x] Every commit is needed on its own, if you have just minor fixes to previous commits, you can squash them.
- [ ] Any new functionality is covered by tests.

### CODE REVIEW

File: themes/pmahomme/scss/_common.scss [hunk 1]
```diff
@@ -92,7 +92,9 @@ input {
     color: #555;
     padding: 4px;
   }
-
+  &:checked[type=checkbox]{
```
Comment (williamdes) [inline themes/pmahomme/scss/_common.scss:95]: Sorry for the review delay, the other themes did not need this fix ?

Comment (ujjwal-batra) [reply themes/pmahomme/scss/_common.scss:95]: @williamdes yeah i Checked for other themes they dknt need this fix.

**Comment by ujjwal-batra** [comment]
@williamdes is there any change required further ?

**Comment by williamdes** [comment]
> @williamdes is there any change required further ?

Nope, I am testing your fix ^^
I will push a rebase and squash commit before merging

**Review by williamdes — CHANGES_REQUESTED** [review]
I am not sure this is the right fix, it seems like `form-check-input` classes are the problem here

**Comment by ujjwal-batra** [comment]
@williamdes okay i will look into it

**Pull request closed without merging**